### DISCUSSION TITLE
Multimodal example

### DISCUSSION THREAD

**Discussion by agatadefr** [Project-MONAI/MONAI#7412] [Ideas]
Hello, 
I  am looking for a tool to perform multimodal analysis traning. Especially radiology + pathology images. Could it be passible within MONAI ? Do you have any example? 
I would appreciate for any info. 
Best,
Agata

**Comment by KumoLiu** [comment]
Hi @agatadefr,
Perhaps it helps.
https://github.com/Project-MONAI/tutorials/tree/main/multimodal/openi_multilabel_classification_transchex
Thanks.
